ixofoundation/ixo-webclient
# fix(submitentityclaim): various bug fixes

## Scope

https://nonaredteam.atlassian.net/browse/IXO-794
https://nonaredteam.atlassian.net/browse/IXO-793

## Work Done

Fixed location selector
Fixed error messages appearing when question loads

## Steps to Test

Submit a claim and see that when you land on question error messages dont appear
* There is one problem with the checkboxes that I couldnt fix though but I dont think this is a big deal for now
